src/Batch/FileConventions/Src/AzureBatchFileConventions/app.config

```diff
@@ -6,6 +6,10 @@
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-6.0.0.0" newVersion="6.0.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.WindowsAzure.Storage" publicKeyToken="31bf3856ad364e35" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-7.2.1.0" newVersion="7.2.1.0" />
```

Comment:
This is extremely unsafe!  Because of breaking changes from 6.x to 7.x this will result in MissingMethodException at runtime - I know this because it's bitten me!